File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/Imperial.java
package com.example.testlunu.net.acu.cityinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Imperial {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/Imperial.kt
package com.example.testlunu.net.acu.cityinfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Imperial(
    @SerializedName("Value")
    @Expose
    var value: Int?,
    @SerializedName("Unit")
    @Expose
    var unit: String?,
    @SerializedName("UnitType")
    @Expose
    var unitType: Int?
)